client/src/research/BubbleChart.js

```diff
@@ -24,8 +26,30 @@ class BubbleChart extends Component{
     };
     var reactNode = <div style={styles} title={`n=${n}`}></div>;
     return reactNode;
+  } 
+  makeNormBubble(percentage, n){
+    var color = (percentage>0)? "black": "green";
+    var dimensions = Math.abs(percentage)*100/65; //resize bubbles: 65% will take up a full bubble. from Milkman   
+    var styles = {
+      backgroundColor:color,
+      height:dimensions,
+      width:dimensions,
+      borderRadius: 100, 
+    };
+    var reactNode = <div style={styles} title={`n=${n}`}></div>;
```

Comment:
As a style change, let's made these variable all `const` instead of `var` (to signify that they are constant and don't change).